VMEncryption/main/Utils/HandlerUtil.py

```diff
@@ -403,72 +408,108 @@ def redo_current_status(self):
                               stat[0]["status"]["code"],
                               stat[0]["status"]["formattedMessage"]["message"])
 
+    def reject_settings(self):
```

Comment:
consider adding a unit test for this function

Reply:
I will check the feasibility of adding a unit test here.

Reply:
I will add unit test for this in another PR.